mleap-tensorflow/src/test/scala/ml/combust/mleap/tensorflow/TensorflowModelSpec.scala

```diff
@@ -44,4 +54,51 @@ class TensorflowModelSpec extends FunSpec {
       }
     }
   }
+
+  describe("with an tensorflow model has variables") {
+    it("saved model") {
+      var reducedSum = 0.0f
+      val testFolder = Files.createTempDirectory("tf-saved-model-export")
```

Comment:
In `TensorFlowModel.scala` you manually call `FileUtil().rmRF` on your temp directory, so I just want to make sure that cleanup is not necessary in this test

Reply:
FileUtil().rmRF(testFolder.toFile) added after we finished using it